MobileCenterAnalytics/MobileCenterAnalytics/Support/MobileCenterAnalytics.xcconfig

```diff
@@ -1,3 +1,3 @@
 #include "../../../Global.xcconfig"
 
-OTHER_LDFLAGS = $(inherited) -framework Foundation -framework UIKit -framework CoreTelephony;
+OTHER_LDFLAGS = $(inherited) -framework Foundation -framework CoreTelephony;
```

Comment:
CoreTelephony is one of frameworks that we need to think about before supporting various platforms. Just keep this here for now since we have a separate task to make it work on macOS.